hack/update-depstat.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+source "${KUBE_ROOT}/hack/lib/init.sh"
+source "${KUBE_ROOT}/hack/lib/util.sh"
+
+kube::golang::verify_go_version
+kube::golang::setup_env
+
+
+echo 'installing depstat'
+pushd "${KUBE_ROOT}/hack/tools" >/dev/null
+  GO111MODULE=on go install github.com/RinkiyaKeDad/depstat
```

Comment:
@dims a very weird thing is happening here. After having these two on the same line when I run the update script locally it gives me this error:
```
installing depstat
running depstat
2021/04/07 15:37:47 exit status 1
```

But when I have them on separate lines then it works properly and generates the JSON file.

Reply:
@RinkiyaKeDad: verify script is reporting
```
diff: /home/prow/go/src/k8s.io/kubernetes/dependency-stats.json: No such file or directory
2021/04/07 09:30:06 exit status 1
```
ref [logs](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/100792/pull-kubernetes-verify/1379725815477964800#1:build-log.txt%3A799)

it means it didn't find file `dependency-stats.json` to compare with

also noticed the `./hack/update-depstat.sh` file is not executable

run this
```
chmod +x ./hack/update-depstat.sh
./hack/update-depstat.sh
```
and update the PR with changed files and `dependency-stats.json`

Reply:
also run
```
chmod +x ./hack/verify-depstat.sh
```

Reply:
there are 4 other failing tests in `/test pull-kubernetes-verify` 
you can reproduce them locally by running `make verify`
- for boiler plate test, update both script header as suggested in `hack/boilerplate/boilerplate.sh.txt`, take a look at other hack/verify scripts
- for internal modules, run `./hack/update-internal-modules.sh` and commit the changes

you should always run `make update` (commit any changes) and `make verify` before creating/updating the PR

Reply:
Hi @navidshaikh! 

> it means it didn't find file `dependency-stats.json` to compare with

Yes, that's why I was trying to run the update script locally so that it generates the json file. But now I'll just generate the file by running depstat on my fork.



> also noticed file `./hack/update-depstat.sh` is not executable

I ran the `chmod` commands you mentioned on both the script files but still wasn't able to run them locally.